tests/test_utils.py

```diff
@@ -561,3 +561,30 @@ def parse(content):
     assert ColorInterp.alpha in color
     assert ColorInterp.red not in color
     assert ColorInterp.gray in color
+
+
+def test_get_array_statistics_coverage():
+    """Test statistics with coverage array."""
+    # Data Array
+    # 1, 2
+    # 3, 4
+    data = np.ma.array((1, 2, 3, 4)).reshape((1, 2, 2))
+
+    # Coverage Array
```

Comment:
Maybe we should call this the `weight` array instead of `coverage`? `coverage` doesn't seem to make the most sense to me here.

Reply:
in https://github.com/isciences/exactextract, weight are really weight, while the `coverage` is called `cell coverage fractions`. I didn't want to confuse people 🤷

Reply:
I see, `coverage` came from exactextract because their weights are usually derived from the coverage of each polygon in the cell. I think `weight` is more general than `coverage` though. There could be use cases for weighted zonal stats that aren't partial-coverage

Reply:
`coverage_weights` ?

Reply:
Would we ever have both `coverage` and `weights`?

Reply:
It looks like exactextract does support both `coverage` and `weight` in effect, because it allows `weight` as a parameter and it computes `coverage` itself

Reply:
> Would we ever have both coverage and weights?

Not planned right now but I don't want to be blocked in the future. I agree that `coverage` is not a perfect name but I don't want to use `weights` because it's not it's not `weights` but spatial fraction. I'm open to change to better name if you have ideas :D